### DISCUSSION TITLE
proxying vimeo player redirects to vimeo home page

### DISCUSSION THREAD

**Discussion by etizaz98** [chimurai/http-proxy-middleware#798] [Q&A]
### Checks

- [X] I understand project setup issues should be asked on [StackOverflow](https://stackoverflow.com/questions/tagged/http-proxy-middleware) or in [GitHub Discussions](https://github.com/chimurai/http-proxy-middleware/discussions).
- [X] I updated to latest `http-proxy-middleware`.

### Describe the bug (be clear and concise)

 have the embed links for vimeoplayer i want to use my express server as proxy. I have added the endpoint which bascially proxies to vimeo embed link. My code is working fine on my local machine but on server sometimes it redirects to vimeo home page(vimeo.com).I am using http-proxy-middleware npm package and have done docker deployment using kubernetes on server. kindly let me know what steps can i take to stop it from redirecting and my proxy behave as it is intended to behave

Code is attached

app.use(`/test-vimeo`, proxy({target: `https://player.vimeo.com/video/709386126?h=21417feddb`,changeOrigin: true}))

### Step-by-step reproduction instructions

```shell
run the proxy in docker
```

### Expected behavior (be clear and concise)

it should proxy vimeo player link

### How is http-proxy-middleware used in your project?

```shell
npm ls http-proxy-middleware
```

### What http-proxy-middleware configuration are you using?

```typescript
i am using version 2.0.0
```

### What OS/version and node/version are you seeing the problem?

```shell
I am using Mac OS
```

**Comment by chimurai** [comment]
Can you provide a reproducible example?
